os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -251,3 +251,31 @@ config AMEBASMART_NP_LP_CHECK_INTERVAL
 		Interval for hardware timer to check KM4/KM0 status
 
 endmenu
+
+menu "Realtek RTL8730E KM4 IPC Logging"
+	
+config RTL8730E_KM4_LOGTASK_PRIO
```

Comment:
1. RTL8730E configs are in os/board/rtl8730e/Kconfig, not here.
2. Code also is in rtl8730e, not amebasmart.

Let's move them there.